test/CMakeLists.txt

```
@@ -0,0 +1,50 @@
+cmake_minimum_required (VERSION 3.8)
+
+unset(CMAKE_ARCHIVE_OUTPUT_DIRECTORY)
+unset(CMAKE_LIBRARY_OUTPUT_DIRECTORY)
+unset(CMAKE_RUNTIME_OUTPUT_DIRECTORY)
+
+# A macro that adds an api or cli test
+macro (add_app_test test_filename test_alternative)
+
+    file (RELATIVE_PATH source_file "${CMAKE_SOURCE_DIR}" "${CMAKE_CURRENT_LIST_DIR}/${test_filename}")
+
+    get_filename_component (target "${source_file}" NAME_WE)
+
+    add_executable (${target} ${test_filename})
+
+    target_link_libraries (${target} "${PROJECT_NAME}_lib" seqan3::seqan3 gmock gmock_main gtest pthread)
+
+    # Add main application as a dependency for command line tests
+    if (test_alternative STREQUAL "CLI_TEST")
```

This would always evaluate to false, because the variable needs to be accessed with `${ }`:
```suggestion
    if (${test_alternative} STREQUAL "CLI_TEST")
```